[2024-11-18T21:50:05.027+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-18T21:50:05.092+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T21:40:00+00:00 [queued]>
[2024-11-18T21:50:05.131+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T21:40:00+00:00 [queued]>
[2024-11-18T21:50:05.134+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-18T21:50:05.206+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_api_sales_data> on 2024-11-18 21:40:00+00:00
[2024-11-18T21:50:05.225+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4775) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-18T21:50:05.229+0000] {standard_task_runner.py:72} INFO - Started process 4777 to run task
[2024-11-18T21:50:05.230+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'raw_api_sales_data', 'fetch_api_sales_data', 'scheduled__2024-11-18T21:40:00+00:00', '--job-id', '468', '--raw', '--subdir', 'DAGS_FOLDER/raw/__pycache__/fetch_data.py', '--cfg-path', '/tmp/tmp6mi54oct']
[2024-11-18T21:50:05.238+0000] {standard_task_runner.py:105} INFO - Job 468: Subtask fetch_api_sales_data
[2024-11-18T21:50:05.346+0000] {task_command.py:467} INFO - Running <TaskInstance: raw_api_sales_data.fetch_api_sales_data scheduled__2024-11-18T21:40:00+00:00 [running]> on host c06adc74012b
[2024-11-18T21:50:05.994+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_api_sales_data' AIRFLOW_CTX_TASK_ID='fetch_api_sales_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T21:40:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T21:40:00+00:00'
[2024-11-18T21:50:05.995+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-18T21:50:05.996+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-18T21:50:05.996+0000] {logging_mixin.py:190} INFO - Current task name:fetch_api_sales_data state:running start_date:2024-11-18 21:50:05.095205+00:00
[2024-11-18T21:50:05.997+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data and current dag run status:running
[2024-11-18T21:50:05.998+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-18T21:50:07.656+0000] {fetch_data.py:29} INFO - Successfully fetched sales data from the API.
[2024-11-18T21:50:07.712+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-18T21:50:07.747+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-18T21:50:07.748+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=raw_api_sales_data, task_id=fetch_api_sales_data, run_id=scheduled__2024-11-18T21:40:00+00:00, execution_date=20241118T214000, start_date=20241118T215005, end_date=20241118T215007
[2024-11-18T21:50:07.777+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-18T21:50:07.778+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-18T21:50:07.779+0000] {logging_mixin.py:190} INFO - Dag name:raw_api_sales_data queued_at:2024-11-18 21:50:01.074733+00:00
[2024-11-18T21:50:07.781+0000] {logging_mixin.py:190} INFO - Task hostname:c06adc74012b operator:PythonOperator
[2024-11-18T21:50:07.822+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-18T21:50:07.920+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-18T21:50:07.937+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
